cluster/internal/data.proto

```diff
@@ -69,3 +69,15 @@ message ExpandSourcesResponse {
     required bytes  Sources  = 1;
     optional string Err      = 2;
 }
+
+message RemoteMonitorRequest {
+    required string RemoteAddr = 1;
```

Comment:
Might want to make this a repeated field so it can retry a few endpoints if one fails.